### DISCUSSION TITLE
DownloadHelper no longer works for me since the last update (10th of january 2024)

### DISCUSSION THREAD

**Discussion by Mikka28** [aclap-dev/video-downloadhelper#75] [Help and Q&A]
### Your question

Hi

I have been using DownloadHelper for years and it always worked for me, but since the automatic update on the 10th of janury 24, it no longer works at all.
I can't donload any videos anymore.

I have tried to download the CoApp, but it doesn't work.
Every time I try, the installation tells me it failed, but it doesn't tell me why
Or how to fix it
Or what the issue is
Just that it failed
I also tried installing the Video DownloadHelper Companion App 1.6.3 (both pgk and dmg) and that works for me.
It does successfully install
But even after I did that and it worked.
The CoApp itself still can not be installed. It still says that a failure occured, but not what the issue is.

My Browser: Firefox 115.7.0esr (64-Bit)
OS:  Mac (High Sierra 10.13.6)
I tried to install this one (Download for MacOS 10.13 & 10.14): https://www.downloadhelper.net/w/CoApp-Installation
together with: Video DownloadHelper Companion App 1.6.3: https://www.downloadhelper.net/install-coapp-v1

I don't know what that means:
    
    On Mac, the output of /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp --info

Is it this?

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: ---
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: no.

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: ---
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: no.

**Comment by paulrouget** [comment]
Can you update to MacOS 10.14 ?

**Reply by Mikka28** [reply]
How does that help?

-> I want to understand what the problem is

**Reply by paulrouget** [reply]
The error you see is because our app is not compatible with the oldest version of OSX.

I managed to fix the coapp for 10.14, and I assumed it would work with 10.13 too. But I was obviously wrong.

Supporting old version of MacOS has been difficult. High Sierra (10.13) hasn't been supported by Apple for 4 years now.

**Reply by Mikka28** [reply]
You made the CoApp Installation for the older macs?
That is so cool!
Thank you!

O.K. I guess I have to upgrade
I hoped I could get around that because apparently there are some compatibility issues with iMovie
So I first have to move iMovie to an extrenal hardrive which will take a few weeks.

**-> So, if I upgrade to Mojave 10.14 your CoApp will work for me?**

**Reply by paulrouget** [reply]
> So, if I upgrade to Mojave 10.14 your CoApp will work for me?

In theory, yes. If it doesn't, I'll assist you to get it to work.

**Reply by Mikka28** [reply]
O.K.
I'll be back in a few weeks, once I have sorted out things (need to sort out the money to get an external hardrive and configure it etc ... )
Thank you, again.

**Reply by petzl20** [reply]
Could you consider supporting 10.13.6?   
Yes, its older, but there are a lot of us here at 10.13.6, who cannot upgrade, 
due to hardware limitations (and for whom VDH is no longer functional).

**Comment by mkft381** [comment]
I have the same issue but cannot upgrade to 10.14. Is there any hope for me?

**Reply by kathrynmont** [reply]
Sorry to bother you all, but I'm soooo confused by all this!  I've happily used this app for years with rare problems but since the changes made in January, it's been almost impossible to use (for example, taking almost 3 hours to download a 45min or so video from a site I'd never before had problems with).  At any rate, I've spent almost 8 hours today trying to figure out what to do and hope someone might point me in the right direction!?

I have the same issue -- though with Windows 10.  For some reason and in order to get it to work, I have had to purchase two $28.50 license fees over the years -- now getting a message that I need to buy another one?  I have updated my Firefox browser 3 times today and used the online download link for Windows10.  
The error message I'm receiving is =>
Exit code: 1 
moz-extension://3ff884cf-bb74-451a-a5e9-bd658aa7fd6e/background/main.js:1
receive@moz-extension://3ff884cf-bb74-451a-a5e9-bd658aa7fd6e/background/main.js:1:13157
Jp</callCatchAppNotFound/</<@moz-extension://3ff884cf-bb74-451a-a5e9-bd658aa7fd6e/background/main.js:1:42564

Are there two versions of the coApp and if so, how do I get the one my browser will read??


Also, here is some info I found about the latest coApp I'd purchased =>
version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only first 16 fisrt characters): f61c635ba6844644
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\kmont",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}v

Thanks very much ...
Kathryn

**Reply by paulrouget** [reply]
Install the latest version of the coapp: https://www.downloadhelper.net/w/CoApp-Installation

**Reply by kathrynmont** [reply]
Omigosh, Paul ... THANK YOU!! I think it's working now!  I'd downloaded to my desktop from that link a number of times today, but guessing I'd been using an old link on the desktop when installing.  Thanks so very much -- you made my day! :)

**Reply by mkft381** [reply]
The installation fails every time.

**Reply by Mattmjs** [reply]
@mkft381 , why not upgrade to 10.14 ? the hardware would be compatible or not ?
If it's not an option then you'd have to stay on old versions of the extension and companion app as well as disable updates.
Or what about dual booting Linux, surely Macs support this. I know it can be done on Windows.
If you can dual boot Linux on Mac, you can keep Mac os but then on Linux install new extension and companion apps.

**Reply by petzl20** [reply]
@Mattmjs Some people cannot upgrade to 10.14.  
Some people are stuck at 10.13.6 High Sierra, and cannot now use Videodownhelper, apparently.  
I'm also one of those ppl.  VDH worked, wonderfully, for me. Now, it does not work.
